src/javaCourse/homework/lesson8/Order.java

```diff
@@ -0,0 +1,70 @@
+package javaCourse.homework.lesson8;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+public class Order implements Serializable {
+    private final int orderId;
```

Comment:
propose to add serial version uid https://www.baeldung.com/java-serial-version-uid

Reply:
Thanks for the heads-up about it. This is a good catch! 
I'll take a look at the article you shared and implement the changes accordingly later on.